src/main/kotlin/no/nav/klage/oppgave/domain/view/OppgaveView.kt

```diff
@@ -13,10 +13,15 @@ data class OppgaveView(
     val ytelse: String,
     val hjemmel: List<String>,
     val frist: LocalDate?,
-    val saksbehandler: String
+    val saksbehandler: Saksbehandler? = null
 ) {
     data class Bruker(
         val fnr: String,
         val navn: String
     )
+
+    data class Saksbehandler(
+        val ident: String,
```

Comment:
Det er kanskje litt coding for the future, men kunne det vært et poeng å hente ut epostadressen her også? Da har vi IDen som brukes i Azure hvis vi trenger å gjøre noe mer, samt at det kan være greit å vise frem epost-adressen i gui ofte også.

Reply:
Merger slik det er nå, så skal jeg se om jeg finner ut hva det feltet hette :)